ampbench_cli

```diff
@@ -122,11 +137,45 @@ function analyze(url) {
             },
             sd_validation: make_api_sd_validation(api_validate_sd_return)
         }
-    };
+        };
 
+    //Logic here
     let write_api_response = () => {
-        console.log(make_api_response());
-    };
+
+        make_api_response();
+
+        switch ('FAIL'){
+
+            case amp_status:
+                var message = "Check the errors at: http://ampbench.appspot.com/api2?url=" + url + " ";
+                var state = criticalThreshold;
+                o.nagiosExit(state, message);
+
+            case api_validate_amp_warnings_return.amp_val_results_status:
+                var message = "Check the errors at: http://ampbench.appspot.com/api2?url=" + url + " ";
+                var state = criticalThreshold;
+                o.nagiosExit(state, message);
+
+            case check_google_amp_cache_return.check_google_amp_cache_status:
+                var message = "Check the errors at: http://ampbench.appspot.com/api2?url=" + url + " ";
+                var state = criticalThreshold;
+                o.nagiosExit(state, message);
+
+            case check_robots_txt_return.check_robots_txt_status:
+                var message = "Check the errors at: http://ampbench.appspot.com/api2?url=" + url + " ";
+                var state = criticalThreshold;
+                o.nagiosExit(state, message);
+
+            case make_api_sd_validation(api_validate_sd_return):
+                var message = "Check the errors at: http://ampbench.appspot.com/api2?url=" + url + " ";
```

Comment:
Sure that this is working? make_api_sd_validation returns and structure, not an String.
I would prefer something more additive. Now, if the first fails, we do not check the rest.